RELEASE.md

```diff
@@ -58,88 +59,129 @@ example, any of the following should trigger a major version increment:
-     request].
+    1. Open a [pull request] from your `release-` branch to merge into `master`.
+       List the key changes for a release in the pull request description. (The
+       diff will show you exactly what has changed since the previous release.)
+       See [the v1.0.0 pull request](https://github.com/18F/web-design-standards/pull/1726)
+       for an example.
 
-  1. Write the release notes on GitHub:
+    1. Tag the release on the `master` branch **or** create the tag when you
+       draft the release notes.
 
-    1. [Draft the release][draft release] from the corresponding tag on the
-       `master` branch.
+    1. Merge the release commits back into `develop` from `master` with a [pull
+       request].
 
-    1. Have at least one team member review the release notes.
+    1. Write the release notes on GitHub:
 
-    1. Publish the [release](https://github.com/18F/web-design-standards/releases)
-       on GitHub.
+        1. [Draft the release][draft release] from the corresponding tag on the
+           `master` branch.
 
-  1. Update the docs site with the new version number and release notes:
+        1. Have at least one team member review the release notes.
 
-    1. Update the `uswds` Node dependency to the new version, e.g.:
+        1. Publish the [release](https://github.com/18F/web-design-standards/releases)
+           on GitHub.
 
-      ```sh
-      cd path/to/web-design-standards-docs
-      export VERSION=1.0.0
-      git fetch origin
-      git checkout -b release-${VERSION} origin/develop
-      npm install --save-dev uswds@${VERSION}
-      ```
+    1. Update the docs site with the new version number and release notes:
+
+        1. Update the `uswds` Node dependency to the new version, e.g.:
 
-    1. Update the `version` [variable in
-       _config.yml](https://github.com/18F/web-design-standards-docs/blob/master/_config.yml#L3).
+            ```sh
+            cd path/to/web-design-standards-docs
+            export VERSION=1.0.0
+            git fetch origin
+            git checkout -b release-${VERSION} origin/develop
+            npm install --save-dev uswds@${VERSION}
+            ```
 
-    1. Follow the above release process to merge the changes to `master` via a
-       [pull request on the docs repo](https://github.com/18F/web-design-standards-docs/compare),
-       minus the GitHub release notes.
+        1. Update the `version` [variable in
+           _config.yml](https://github.com/18F/web-design-standards-docs/blob/master/_config.yml#L3).
 
+        1. Follow the above release process to merge the changes to `master` via a
+           [pull request on the docs repo](https://github.com/18F/web-design-standards-docs/compare),
+           minus the GitHub release notes.
+
+### Pre-releases
+
+When releasing potentially disruptive changes, it's good practice to publish pre-releases of
+planned versions. These are sometimes also called [release candidates]. Here's how it works:
```

Comment:
release candidates isn't linking to anything here. I think because it's defined as a singular at the bottom of the doc.